presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotBrokerPageSource.java

```diff
@@ -280,8 +281,18 @@ private int issuePqlAndPopulate(
 
     private static String asText(JsonNode node)
     {
+        if (node.isNull()) {
```

Comment:
Where is this logic around rendering a json array to string used in this PR ?

Reply:
I think this is to show  multi-value column as string.

Reply:
So is this a related change ? Where do multi value columns creep in here ? Or should this be a different PR ?

Reply:
I was test with a table has multi-value column, hence this change, I can revert it and open another pr.